payment_cycle/gql_mutations.py

```diff
@@ -0,0 +1,23 @@
+import graphene
+from django.core.exceptions import PermissionDenied
+from django.utils.translation import gettext as _
+
+from core.schema import OpenIMISMutation
+from payment_cycle.apps import PaymentCycleConfig
+from payment_cycle.services import PaymentCycleService
+
+
+class ProcessBenefitPlanPaymentCycleMutation:
```

Comment:
Current definition of class causes issue with not displaying that mutation on graphQL level therefore it's not visible.   
it should be: 
`class ProcessBenefitPlanPaymentCycleMutation(BaseMutation):` ?  Can you check this?

Reply:
i added OpenIMISMutation base class